src/iqm/qiskit_iqm/fake_backends/iqm_fake_backend.py

```diff
@@ -221,8 +223,13 @@ def _create_noise_model(
         """
         Builds a noise model from the attributes.
         """
+
         noise_model = NoiseModel(basis_gates=["r", "cz"])
 
+        if architecture.name == "Deneb":
+            # noise_model.add_basis_gates(["move"]) does not work because it checks against a list of standard gates
+            noise_model = NoiseModel(basis_gates=["r", "cz", "move"])
+
```

Comment:
Get the basis gates from the architecture, so it also works for future devices.
```suggestion
        iqm_to_qiskit_gates = {iqm: qiskit for iqm, qiskit in IQM_TO_QISKIT_GATE_NAME}
        for iqm_gate in architecture.operations.keys():
            if iqm_gate not in ["measure", "barrier"] + iqm_to_qiskit_gates.keys():
                iqm_to_qiskit_gates[iqm_gate] = iqm_gate
        noise_model = NoiseModel(basis_gates=iqm_to_qiskit_gates.values())
```

Reply:
Which would require the use of IQM_TO_QISKIT_GATE_NAME below to use iqm_to_qiskit_gates instead.